doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
+# About networking
+
+There are different ways to connect your instances to the Internet. The easiest method is to have LXD create a network bridge during initialization and use this bridge for all instances, but LXD supports many different and advanced setups for networking.
+
+## Network devices
+
+Every instance must be assigned at least one network device, also called {abbr}`NIC (Network Interface Controller)`.
```

Comment:
> Every instance must be assigned at least one network device

This isn't true.

Reply:
In fact you may want to explicity configure an instance with no NIC devices and then use the `proxy` device to provide inbound/outbound services without allowing the instance general network access.